### DISCUSSION TITLE
cruise_mode.py Throws

### DISCUSSION THREAD

**Discussion by NlGHTLORD** [WhiteMagic/JoystickGremlin#529] [Q&A]
When I try to load up the script [cruise_mode.py](https://github.com/WhiteMagic/JoystickGremlinModules/blob/master/cruise_mode.py), I get the following error:

AttributeError: module 'gremlin.actions' has no attribute "Tempo"

I'm sure I'm missing something simple if someone can point out what though I would be grateful.

Yogi got rid of cruise control in SC so now, I'm looking for a work around.

**Comment by WhiteMagic** [comment]
The issue with that plugin is that it's from before the new plugin system was introduced, and so it is outdated in many different ways. One is the tempo container, but also the definition of devices has changed since then. For this to be usable again it'd have to be rewritten in the new plugin system and checked what aspect about the Tempo container has changed since.
